content/pages/de/medien/2014_11_18_dienst_uepf_und_ddr_einsicht.md

```diff
@@ -0,0 +1,32 @@
+Title: Dienst ÜPF und DDR: Einsicht ist der erste Schritt zur Besserung!
+Slug: 2014_11_18_dienst_uepf_und_ddr_einsicht
+Lang: de
+Date: 2014-11-18
+
+Liebe Medienschaffende 
+
+Mit Wohlwollen begrüsst der Chaos Computer Club Schweiz CCC-CH die Haltung der Mitarbeitenden des Überwachungsdienstleisters des Bundes "ÜPF". Die Dienst ÜPF-Mitarbeitenden selbst haben in ihren Büroräumen nicht nur die Schweizer Flagge gehisst, sondern zudem die eines für seine totale Überwachung bekannten pseudo-demokratischen ehemaligen ostdeutschen Staates – die der DDR nämlich. Hier gibt es ein Foto davon: 
+
+![](images/schweiz_und_ddr.jpg)
```

Comment:
Could you add an alt text in here, for people using screenreaders? Maybe something like this:

    ![Flagge der Schweiz und der DDR hängen nebeneinander](images/schweiz_und_ddr.jpg)